### PR TITLE
feat: add support for SQLAlchemy 1.4

### PR DESCRIPTION
**Pull request by saartochner**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -4,13 +4,15 @@ mock
 boto3
 pytest-cov==2.6.1
 attrs==19.1.0
-requests==2.28.1
```
Comment (saartochner) [inline requirements.txt:7]: most of the changes are depandabot requests

File: src/lumigo_tracer/wrappers/sql/sqlalchemy_wrapper.py [hunk 2]
```diff
@@ -73,16 +73,14 @@ def _handle_error(context):  # type: ignore[no-untyped-def]
 def execute_wrapper(func, instance, args, kwargs):  # type: ignore[no-untyped-def]
     result = func(*args, **kwargs)
     with lumigo_safe_execute("sqlalchemy: listen to engine"):
-        listen(result, "before_cursor_execute", _before_cursor_execute)
-        listen(result, "after_cursor_execute", _after_cursor_execute)
-        listen(result, "handle_error", _handle_error)
+        listen(instance, "before_cursor_execute", _before_cursor_execute)
+        listen(instance, "after_cursor_execute", _after_cursor_execute)
+        listen(instance, "handle_error", _handle_error)
     return result
```
Comment (therightstuff) [inline src/lumigo_tracer/wrappers/sql/sqlalchemy_wrapper.py:79]: is it still correct to return `result`?

**Review by therightstuff — APPROVED** [review]
i have a question, but it LGTM

File: src/lumigo_tracer/wrappers/sql/sqlalchemy_wrapper.py (hunk 2 above)
Comment (saartochner) [reply src/lumigo_tracer/wrappers/sql/sqlalchemy_wrapper.py:79]: yes :)
(Though results should always be None...)

**Pull request merged**